### ISSUE TITLE
Analyze Phenotypes: Clarify that profile input is phenotypes only, not diseases

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1296]
It is not clear what it means or should mean that certain diseases can be selected (eg. Dementia), but that certain others can not be selected (eg. Charcot-Marie...). 

Moreover, what the expected behavior is once those diseases are selected is also not clear. 

Parking this here for discussion

**Comment by jmcmurry** [comment]
Just realized that here Dementia is both a disease and a phenotype. As they are both HP terms, it was confusing to me. Not sure how to make this more apparent. Changing the title of the issue accordingly.

**Comment by jmcmurry** [comment]
Results like this make it especially confusing.
This looks like dementia disease versus dementia phenotype when in fact it is dementia phenotype in two different taxa.

<img width="364" alt="screen shot 2016-07-11 at 11 10 32 am" src="https://cloud.githubusercontent.com/assets/3658863/16741448/280f2724-4758-11e6-97d1-b078dd687948.png">

**Comment by harryhoch** [comment]
I think his might be an ontology issue. At one point, there had been no call that would let us easily restrict the search to (say) hpo vs. something else .Is this no still the case?

**Comment by jmcmurry** [comment]
The former is HPO, the latter is NBO (eg. Neuro Behavior Ontology, used in ZFIN). I think these are legitimately different results. Will be sorted out once we have taxon options available.

**Comment by iamkoehler** [comment]
I am not sure what you are trying to achieve, but 
- NBO is used in hp and mp as well
- pretty sure users also want to enter a "disease" such as alzheimer, instead of entering every symptom of alzheimer

**Comment by jmcmurry** [comment]
> - pretty sure users also want to enter a "disease" such as alzheimer, instead of entering every symptom of alzheimer

Agreed, this is functionality I described in other tickets (https://github.com/monarch-initiative/monarch-app/issues/1250 and [others](https://github.com/monarch-initiative/monarch-app/labels/Analyze%20Phenotypes)). However, this is what we have presently, so worth making sure that the users expectations are appropriately managed. 

See also https://github.com/monarch-initiative/monarch-app/issues/994
